openwisp_notifications/tests/test_admin.py

```diff
@@ -205,7 +205,7 @@ def test_notification_setting_inline_organization_formfield(self):
         response = self.client.get(
             reverse('admin:openwisp_users_user_change', args=(self.admin.pk,))
         )
-        self.assertContains(response, '<option value="default">Default Type</option>')
+        self.assertContains(response, 'default</option>')
```

Comment:
(This  `'default</option>'` works as well and also saves one db query):

Alternate option :point_down: 
```python
Organization = swapper_load_model('openwisp_users', 'Organization')
....
....
....
self.assertContains(response, '<option value="{Organization.objects.first().id}">default</option>')
```

Reply:
I think this way we are making the test fragile. We should atleast check for organization id in the HTML response.

Reply:
> I think this way we are making the test fragile. We should atleast check for organization id in the HTML response.

So should I go with this  method? @pandafy  👇
```py
Organization = swapper_load_model('openwisp_users', 'Organization')
....
....
....
self.assertContains(response, '<option value="{Organization.objects.first().id}">default</option>')
```